README.md

```diff
@@ -55,13 +55,13 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
       - ./standard-1.yaml
     dependencies:
       certifications: # An optional list of certifications stored remotely
-        - url: github.com/18F/LATO
+        - url: https://github.com/18F/LATO
```

Comment:
Ah, it seems to be here: https://github.com/18F/GSA-Certifications/blob/master/LATO.yaml